invenio_app/ext.py

```diff
@@ -44,13 +48,22 @@ def init_app(self, app, **kwargs):
         """
         # Init the configuration
         self.init_config(app)
-        # Enable Rate limiter
-        self.limiter = Limiter(app, key_func=get_ipaddr)
+
         # Enable secure HTTP headers
         if app.config['APP_ENABLE_SECURE_HEADERS']:
             self.talisman = Talisman(
                 app, **app.config.get('APP_DEFAULT_SECURE_HEADERS', {})
             )
+
+        # Enable Rate limiter
+        # Flask limiter needs to be initialised after talisman, since if the
+        # talisman preprocessor doesn't run you will get an error on it's
+        # afterprocessor.
+        self.limiter = Limiter(
+            app,
+            key_func=app.config.get('FLASK_LIMITER_METHOD') or
```

Comment:
What is this variable? Flask-Limiter's variables are prefixed with ``RATELIMIT_``

Reply:
It's a custom config variable to allow overriding the ```key_func``` method.

Reply:
I have changed it adding a ```RATELIMIT_``` prefix adding a description in the config.